packages/react-components/src/components/InputBoxComponent.tsx

```diff
@@ -39,10 +39,13 @@ import { useTheme } from '../theming';
 import { MentionLookupOptions, _MentionPopover, Mention } from './MentionPopover';
 /* @conditional-compile-remove(mention) */
 import { useDebouncedCallback } from 'use-debounce';
+import { isEnterKeyEventFromCompositionSession } from './utils/keyboardNavigation';
```

Comment:
Could you clarify if there will be a separate PR for moving out the mentions functionality out of `InputBoxComponent`?